OWNERS_ALIASES

```diff
@@ -54,7 +54,7 @@ aliases:
     - ahrtr
     - jmhbnz
     - serathius
-    - wenjiaswe
+    - siyuanfoundation
```

Comment:
@ivanvc we should do this to you as well. Do we want to use this one ticket or a different one?

Will let you two decide: @ivanvc @siyuanfoundation

Reply:
I'll open a pull request myself. Thanks, @wenjiaswe.